README.md

```diff
@@ -170,51 +184,13 @@ A number of callback functions are fired at various points of the flow. The most
     containerId: 'onfido-mount',
```

Comment:
@hjheath we probably don't need the ID though in this example. Also might be helpful to write the arguments which are passed to it.